### DISCUSSION TITLE
Should I manually update an entity's `GlobalTransform`?

### DISCUSSION THREAD

**Discussion by nicopap** [bevyengine/bevy#7025] [Q&A]
The `global_vs_local_translation` example uses the `translation_mut` method on `GlobalTransform`. I thought `GlobalTransform` was **meant to never be updated manually**, why is there a `_mut` method on it then? And why is it manually updated in the example? It seems `translation_mut` is the only mutable method on `GlobalTransform`.

**Comment by rparrett** [comment]
There was some discussion about this a while back here: https://github.com/bevyengine/bevy/pull/5139
